.github/workflows/update-standard.yml

```diff
@@ -0,0 +1,31 @@
+name: update-standard
+
+on:
+  push:
+    paths:
+    - 'dicom_standard/*.py'
+  schedule:
+    - cron: '0 0 1 * *'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+      - name: Set up Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: '3.7'
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements.txt
+      - name: Run Makefile
+        run: |
+          make clean -C dicom_standard
+          make updatestandard -C dicom_standard
+          make -C dicom_standard
```

Comment:
I know that currently we store the HTML and generated JSON inside the git repo, but perhaps it would make sense to keep these large generated files out of the repo and instead store them as Github action "artifacts"?

Also, I suspect you were planning on doing this, but once we have generated the new files, maybe we could:
1. detect if anything changes in the files
2. if something did change, we can create new pypi releases automatically

I guess we could also push to npm for people that are not using python!

Reply:
See https://help.github.com/en/actions/configuring-and-managing-workflows/persisting-workflow-data-using-artifacts

Reply:
BTW, this is just an idea for the future---no need to do this in the current PR!

Reply:
For future reference: https://packaging.python.org/guides/publishing-package-distribution-releases-using-github-actions-ci-cd-workflows/